azure-functions-extension-blob/pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[build-system]
+requires = ["setuptools >= 61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "azure-functions-extension-blob"
+dynamic = ["version"]
+requires-python = ">=3.9"
+authors = [{ name = "Azure Functions team at Microsoft Corp.", email = "azurefunctions@microsoft.com"}]
+description = "Blob Python worker extension for Azure Functions."
+readme = "README.md"
+license = {text = "MIT License"}
+classifiers= [
+        'License :: OSI Approved :: MIT License',
+        'Intended Audience :: Developers',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Operating System :: Microsoft :: Windows',
+        'Operating System :: POSIX',
+        'Operating System :: MacOS :: MacOS X',
+        'Environment :: Web Environment',
+        'Development Status :: 5 - Production/Stable',
+    ]
+dependencies = [
+    'azure-functions-extension-base',
+    'azure-storage-blob==12.19.0'
+    ]
+
+[project.optional-dependencies]
+dev = [
+    'mypy',
```

Comment:
Do we have a mypy tests? If not remove this,

Reply:
same for all pipelines

Reply:
Removed